packages/cli/src/ui/AppContainer.tsx

```diff
@@ -970,7 +971,7 @@ Logging in with Google... Please restart Gemini CLI to continue.
   useKeypress(
```

Comment:
also not a new issue but in the interest of reducing tech debt, please move this directly into the dialog that shows the restart message RestartDialog rather than having this logic arbitrarily separated in AppContainer.